pom.xml

```diff
@@ -669,6 +669,7 @@
             instead -->
           <configLocation>src/conf/maven_checks.xml</configLocation>
           <enableRulesSummary>false</enableRulesSummary>
+          <failOnViolation>false</failOnViolation>
```

Comment:
The checkstyle errors will show the details and the codestyle is configured in the maven_checks.xml file. Your IDE should automatically pick it up and hightlight things as well. At least Eclipse does that.